en/setting-up-the-app-in-shopify.md

```diff
@@ -0,0 +1,116 @@
+
+3. Walk through the Onboarding guide. Make sure that you read everything properly. But no worries, if you miss something, you can trigger the Onboarding guide any time in the future (the link is in the footer).
+
+   - Welcome to Sakurabook app!
+   - Prerequisites
+   - 2 reservation types
+   - Dashboard view
+   - Connect app to theme
+   - Click the `Finish` button
+
+   ![Alt text](../img/?raw=true "Sakurabook Onboarding Finish")
+
+4. Now you appeared on the Use case selection screen. You can select either **Rental space reservation** or **Service reservation**. Make a wise choice, it has to suit your business. This cannot be changed in the future. After you select the right use case, click on `Import` button.
+
+   📌 Yes, it cannot be change, but if you really need to change the use case, please get in touch with the support and we'll make a way to fix it for you.
+
+   ![Alt text](../img/?raw=true "Sakurabook Use Case Selection ")
+
+5. New modal window appeared. You can select products to be imported into the Sakurabook app. Select the products you want to see in the app by clicking the checkboxes on the left side. Once you are done, click the `Add` button.
+
+   📌 If you select **Rental space reservation** use case, you should only see products with the `Rental` tag in the import selector. If you select **Service reservation**, you will see only products with the `Service` tag. AddOns can be imported later.
+
+   📌 Have you selected **Rental space reservation**? Skip to step 8.
+
+   ![Alt text](../img/?raw=true "Sakurabook Import Products")
+
+6. Since you selected **Service reservation**, you have now the option to create your first staff member. Click on the `Add Member` button and find yourself on the Staff creation screen.
+
+   ![Alt text](../img/?raw=true "Sakurabook Add Member")
+
+7. Please check out the detailed [Create Staff]() guide, click **Save** in the top right corner. Congratulations. You have created your first staff. Then click on **Dashboard** in the top navigation menu.
+
+   ![Alt text](../img/Screenshot%202022-08-28%20at%2010.56.35.png?raw=true "Sakurabook Edit Staff")
+
+8. Welcome to Sakurabook Dashboard! This is where you can see and manage all your bookings. Check the [Manage Dashboard]() guide for more details.
+
+   ![Alt text](../img/Screenshot%202022-08-28%20at%2010.51.06.png?raw=true "Sakurabook Dashboard")
+
+9. Now we refer to [Usage Guides](usage-guides.md) where you can find details about managing bookings on the dashboard, setting up and changing options of your staff, rental spaces, and services, and changing the location of your business and all the other staff.
+
+10. The very last step on our Sakurabook Admin Setup guide is a selection of your Billing plan. Navigate to Settings (top menu) and click on the `Manage plan` button under the Billing section. You've got several options, see details about [Billing plans](billing-plans.md).
```

Comment:
I miss the information about the impossibility of creating bookings without a subscription.